ui/apps/platform/src/Components/ReportJob/MyLastJobStatusTh.tsx

```diff
@@ -10,26 +10,42 @@ function MyLastJobStatusTh() {
                 <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsMd' }}>
                     <FlexItem>
                         <p>
-                            The status of your last requested job from the{' '}
-                            <strong>active job queue</strong>. An <strong>active job queue</strong>{' '}
-                            includes any requested job with the status of <strong>preparing</strong>{' '}
-                            or <strong>waiting</strong> until completed.
+                            Displays the status of your most recent job, whether it is currently
+                            running or has completed. The possible statuses are:
                         </p>
                     </FlexItem>
+                    <FlexItem>
+                        <p>
+                            <strong>Waiting:</strong>
+                        </p>
+                        <p>Your job is in the queue, waiting for other jobs to be processed.</p>
+                    </FlexItem>
                     <FlexItem>
                         <p>
                             <strong>Preparing:</strong>
                         </p>
-                        <p>Your last requested job is still being processed.</p>
+                        <p>Your job is still being processed.</p>
                     </FlexItem>
                     <FlexItem>
                         <p>
-                            <strong>Waiting:</strong>
+                            <strong>Ready for download:</strong>
+                        </p>
+                        <p>The report is ready and available for download.</p>
+                    </FlexItem>
+                    <FlexItem>
+                        <p>
+                            <strong>Successfully sent:</strong>
                         </p>
+                        <p>The report has been successfully emailed to you.</p>
+                    </FlexItem>
+                    <FlexItem>
                         <p>
-                            Your last requested job is in the queue and waiting to be processed
-                            since other requested jobs are being processed.
+                            <strong>Error:</strong>
                         </p>
+                        <p>There was an issue with your job. Hover to view the error message.</p>
+                    </FlexItem>
+                    <FlexItem>
+                        <p>If no recent jobs are available, &quot;None&quot; will be displayed.</p>
                     </FlexItem>
```

Comment:
The tooltip text is changed to be more correct in regards to `My last job status`.

<img width="1440" alt="Screenshot 2024-10-16 at 9 29 31 AM" src="https://github.com/user-attachments/assets/ecbe0cc3-c439-4cb5-b5aa-167838347efd">

@mansursyed let me know if this works for you. I can make tweaks as necessary.